build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs

```diff
@@ -16,19 +16,47 @@ public class ProcessLogcatTiming : Task
 		[Required]
 		public string ApplicationPackageName { get; set; }
 
+		[Required]
+		public string DefinitionsFilename { get; set; }
+
 		public string ResultsFilename { get; set; }
 
+		Dictionary<string, Regex> definedRegexs = new Dictionary<string, Regex> ();
+		Dictionary<string, string> results = new Dictionary<string, string> ();
+
+		void LoadDefinitions ()
+		{
+			using (var reader = new StreamReader (DefinitionsFilename)) {
+				string line;
+
+				while ((line = reader.ReadLine ()) != null) {
+					int index = line.IndexOf (';');
```

Comment:
Why'd you choose `;` as the separator? I'd think that ` ` (space) or `=` (equal) would make a bit more sense:

    # current
    last;monodroid-timing:\s+(?<message>.*)$

    # vs one of
    last monodroid-timing:\s+(?<message>.*)$
    last=monodroid-timing:\s+(?<message>.*)$

I think I find the ` ` more readable, but the `=` has a nice connotation of "this is the key with an associated regex."